test/functional/test_runner.py

```diff
@@ -27,6 +27,9 @@
 import logging
 import unittest
 
+# Should fail testing and immediately stop if free space falls below.
+MIN_FREE_SPACE = 16 * 1024 * 1024  # Could be larger.
```

Comment:
Instead of a comment "Could be larger" it could make sense to explain why this was chosen or where the magic number is from.

Also, why are you not choosing the value from `constexpr min_disk_space`?